packages/oscal-react-library/src/components/OSCALBackMatter.tsx

```diff
@@ -0,0 +1,207 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Chip from "@mui/material/Chip";
+import Grid from "@mui/material/Grid";
+import Stack from "@mui/material/Stack";
+import { Typography } from "@mui/material";
+import FormatQuoteIcon from "@mui/icons-material/FormatQuote";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+import StyledTooltip from "./OSCALStyledTooltip";
+import { getAbsoluteUrl, guessExtensionFromHref } from "./oscal-utils/OSCALLinkUtils";
+import { OSCALSection, OSCALSectionHeader } from "../styles/CommonPageStyles";
+import { OSCALMarkupLine } from "./OSCALMarkupProse";
+import OSCALAnchorLinkHeader from "./OSCALAnchorLinkHeader";
+import OSCALEditableTextField, { EditableFieldProps } from "./OSCALEditableTextField";
+import { Resource, ResourceLink, BackMatter } from "@easydynamics/oscal-types";
+import { ReactElement } from "react";
+
+export const OSCALBackMatterCard = styled(Card)(
+  ({ theme }) => `
+    margin-top: ${theme.spacing(1)};
+    display: flex;
+    flex-direction: column;
+`
+);
+
+interface TitleDisplayProps {
+  uuid: string;
+  children: React.ReactNode;
+}
+
+function TitleDisplay(props: TitleDisplayProps): ReactElement {
+  const { children, uuid } = props;
+  return (
+    <OSCALAnchorLinkHeader value={uuid}>
+      <Typography variant="subtitle1">{children}</Typography>
+    </OSCALAnchorLinkHeader>
+  );
+}
+
+interface CitationDisplayProps {
+  readonly resource: Resource;
```

Comment:
Although there is only a single property in this interface, I would suggest still providing documentation here.

Reply:
I was wondering about that, but Resource is a documented field on `oscal-types` so I did not document the field.